l1-contracts/scripts/utils.ts

```diff
@@ -54,9 +54,9 @@ export function web3Provider() {
 
 export function getAddressFromEnv(envName: string): string {
   const address = process.env[envName];
-  if (!/^0x[a-fA-F0-9]{40}$/.test(address)) {
-    throw new Error(`Incorrect address format hash in ${envName} env: ${address}`);
-  }
+  // if (!/^0x[a-fA-F0-9]{40}$/.test(address)) {
```

Comment:
?

Reply:
was an artifact of a local deployment, ill revert